<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>3, 5 - Shamir Secret Sharing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Secret</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Share 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Share 2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Combined Share</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit_Secret"/>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="plainTextEdit_s1"/>
   </item>
   <item row="2" column="2">
    <widget class="QPlainTextEdit" name="plainTextEdit_s2"/>
   </item>
   <item row="2" column="3" rowspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit_cs"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Share 3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Share 4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Share 5</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit_s3"/>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="plainTextEdit_s4"/>
   </item>
   <item row="4" column="2">
    <widget class="QPlainTextEdit" name="plainTextEdit_s5"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_create_share">
     <property name="text">
      <string>Create Shares</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_combine_share">
     <property name="text">
      <string>Combine Shares</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
